code/controllers/subsystem/pai.dm

```diff
@@ -56,7 +69,6 @@ SUBSYSTEM_DEF(pai)
 				candidate.savefile_save(usr)
 			if("load")
 				candidate.savefile_load(usr)
-				//In case people have saved unsanitized stuff.
```

Comment:
?

Reply:
Alpharex said to delete it while I was dealing with this file anyway. Apparently unnecessary (honestly it's all spaghetti, I don't  know)

Reply:
Should leave the comment I'd think. @GoldenAlpharex ?

Reply:
Let me check again.

Reply:
I got a feeling that we had something else there at some point, I think that's why. Might be something I removed previously, but I'm not even sure. We can leave the comment there, but like, it doesn't seem to do anything to me.

Like, all we do is making sure that the name length is respected, I think I removed part of the sanitization there in a previous PR. Because shit was getting double-sanitized.

Reply:
Ah no that code is totally sanitization. Not html sanitization, but totally sanitization. Should keep the comment